### DISCUSSION TITLE
Using T. Gui in ssh console session?

### DISCUSSION THREAD

**Discussion by deivid1** [tui-cs/Terminal.Gui#1455] [Q&A] [answered]
Hi, I can only use the Terminal Gui on a Ubuntu desktop terminal window. If I try on ssh session it displays a blank screen.

**Comment by tznind** [comment]
What version of ubuntu? what OS are you using to SSH to the remote machine? what terminal client (or SSH client) are you using to SSH?

**Reply by deivid1** [reply]
Ubuntu 20.04.
Ssh session with Putty 0.74 , using Windows 10

**Reply by tznind** [reply]
Can you try SSH direct with something like powershell?

**Reply by tznind** [reply]
You can also try running with UseSystemConsole (`-usc`) to see if that helps.

**Comment by deivid1** [comment]
ok I will try thanks. With your answer I assume it´s possible to run the Gui ok in non desktop environments, right?

**Reply by tznind** [reply]
That is correct, there should be no issues running Terminal.Gui under any terminal/OS environment including over SSH tunnel.

**Reply by chadouming** [reply]
I can also confirm there is an issue running this over putty. Works fine over powershell ssh though. Not sure what's going on, but we're basically getting an empty screen covering the whole terminal.

**Comment by tznind** [comment, marked as answer]
@chadouming, @deivid1 Please try the following, set Terminal-type string to `xterm-256color` under PuTTY Configuration `Settings->Connection->Data->Terminal-type string`:

![image](https://user-images.githubusercontent.com/31306100/134478251-e6b746eb-aa6a-49ac-be5a-639d997d6b73.png)


@deivid1 can you please rename this issue to indicate that it is a PuTTY specific thing (unless you think that your problem is broader than that app).

**Reply by chadouming** [reply]
Sorry for the delay, this indeed fixed my issue, thanks :)

**Reply by deivid1** [reply]
I can confirm it works. We can close this issue. Thanks!
